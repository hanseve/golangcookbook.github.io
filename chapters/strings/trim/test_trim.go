package main

import (
	"fmt"
	"strings"
)

func main() {
	s := " Hello world  \n\t\r"
	fmt.Printf("%q\n", s)
	fmt.Printf("%q\n", strings.TrimRight(s, " \n\t\r"))
	fmt.Printf("%q\n", strings.TrimSpace(s))
}
