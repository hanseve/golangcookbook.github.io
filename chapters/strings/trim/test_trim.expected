" Hello world  \n\t\r"
" Hello world"
"Hello world"
